downloads/github-actions-cheat-sheet.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub Actions Cheat Sheet
+byline: GitHub Actions give you the flexibility to build automated software development lifecycle workflows. You can write individual tasks, called actions, and combine them to create  custom workflows in your repository. GitHub Actions are automated processes allowing you to build, test, package, release, or deploy any code project on GitHub, but you can also use them to automate any step of your workflow&#58; merging pull requests, assigning labels, triaging issues to name a few.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Workflow Syntax
+Workflow files use YAML syntax, and must have either a .yml or .yaml file extension. You must store workflow files in the `.github/workflows/` directory of your repository.
+
+```yaml
+name: Node CI
+on: push
+  branches:    
+    - 'releases/*'
+jobs:
+  my_build:
+    runs-on: ubuntu-latest
+    needs: my-other-job
+    steps:
+    - uses: actions/checkout@master
+    - name: Say something
+      run: echo "A little bit more action"
+```
+
+### Workflow `name`
+The name of your workflow. GitHub displays the names of your workflows on your repository's actions page.
+
+### `on` Event
+The name of the GitHub event that triggers the workflow. You can provide a single event string, an array of events, or an event configuration map that restricts the execution of a workflow:
+- When using the `push` and `pull_request` events, `branches` and `tags` allow to select or exclude references the workflow will run on, while `paths` specifies which files must have been modified in order to run the workflow.  
+- The `types` keyword enables you to narrow down activity that causes the workflow to run: `created`, `edited`, `deleted`...
+A workflow trigger can also be scheduled:  
+```yaml
+on:
```

Comment:
The schedule example here seems like a non-sequitur since it isn't mentioned in the text above.  Perhaps include an example mentioned therein.  Also, I think it's worth highlighting that you can filter by `path` as well as `paths-ignore` (although not in the same job) . 

If we mention the `schedule` option, consider adding a link to the [syntax explanation](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onschedule)

Reply:
Absolutely. Something disappeard here and new options have been released since it was written. The whole section need a refresh.